core/src/wallet/ripple/RippleLikeAccount.cpp

```diff
@@ -461,5 +461,28 @@ namespace ledger {
             });
         }
 
+        void RippleLikeAccount::isAddressActivated(const std::string &address,
+                                                   const std::shared_ptr<api::BoolCallback> &isActivated) {
+            isAddressActivated(address).callback(getContext(), isActivated);
+        }
+
+        Future<bool> RippleLikeAccount::isAddressActivated(const std::string &address) {
```

Comment:
First thing first GJ here 
I have some suggestions to improve the readability of this method

Reply:
Thanks ! Are we talking about your comment about variable creation ? or something else ?

Reply:
Exactly and the rework about `BigInt`